islandora_compound_object.module

```diff
@@ -157,19 +157,45 @@ function islandora_compound_object_block_view($delta = '') {
 }
 
 /**
- * Implements hook_CMODEL_PID_islandora_object_purged().
+ * Implements hook_form_FORM_ID_alter().
  */
-function islandora_compound_object_islandora_compoundcmodel_islandora_object_purged($pid) {
```

Comment:
Do we still need a hook to handle cases of objects being purged not within the forms?

Reply:
Books and collections both don't handle the case of purging without using the forms. Does compound need that functionality?

Reply:
IMO its better to leave orphaned children with dangling references, then to remove all the references and make them parentless, as the orphan report can at least find them if the have dangling references, but its possible there is a use case I don't see here.

Reply:
Regarding this, i feel the hook idea is good, but would be more effective to have something in islandora itself to deal with that need/any other module that leaves orphans behind. Personally this pull as it is now solves the main issue of removing/deleting via forms, which is a good start? @jordandukart did you had time to test this/want to review? Don't want to step on people's toes by pushing forward.

Reply:
I don't really have an opinion for or against this behavior was just pointing out a change that -could- effect someone's current assumptions.